Components/NativeMethods.cs

```diff
@@ -858,6 +858,11 @@ public enum ExtendedWindowStyles : uint
 
         [DllImport("psapi.dll", BestFitMapping = false)]
         public static extern uint GetProcessImageFileName(IntPtr hProcess, [Out] StringBuilder lpImageFileName, [In][MarshalAs(UnmanagedType.U4)] int nSize);
+        
+        [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Auto)]
```

Comment:
ccv
```suggestion
        [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.AutOOOO)]
```
yoooo

fghdfg